redbot/setup.py

```diff
@@ -385,7 +394,12 @@ async def remove_instance_interaction() -> None:
         "Note: Choosing PostgreSQL will prevent the setup from being completely non-interactive."
     ),
 )
-@click.option("--overwrite-existing-instance", type=bool, is_flag=True)
+@click.option(
+    "--overwrite-existing-instance",
+    type=bool,
+    is_flag=True,
+    help=("Overwrites an existing instance."),
```

Comment:
Maybe add a note that this can erase the bot? Should be obvious but given #support....

Reply:
I agree with this idea.